js/index.js

```diff
@@ -24,6 +24,7 @@ const urlAdds = {
 function renderFeatures(data) {
   if(Array.isArray(data)) {
     let tmp = data.join(", ");
+    tmp = tmp.replaceAll("\nsrl", " srl");
```

Comment:
@siva19susi 
Can we replace `\n` with ``? Because there will be cases where features list will be as follows

```
not srl_nokia-feat:platform-7220-h2\n        srl_feat:something`
```